include/kodlab_mjbots_sdk/lcm_subscriber.h

```diff
@@ -1,85 +1,102 @@
-// BSD 3-Clause License
-// Copyright (c) 2021 The Trustees of the University of Pennsylvania. All Rights Reserved
-// Authors:
-// Shane Rozen-Levy <srozen01@seas.upenn.edu>
+/**
+ * @file lcm_subscriber.h
+ * @author Ethan Musser (emusser@seas.upenn.edu)
+ * @author J. Diego Caporale (jdcap@seas.upenn.edu)
+ * @author Shane Rozen-Levy (srozen01@seas.upenn.edu)
+ * @brief Provides LCM subscriber object for subscribing to onw or multiple LCM
+ * channels.
+ * @date 7/7/22
+ *
+ * @copyright Copyright 2022 The Trustees of the University of Pennsylvania. All
+ * rights reserved. BSD 3-Clause License
+ *
+ */
 
 #pragma once
+
+#include <map>
+#include <mutex>
 #include <string>
 #include "lcm/lcm-cpp.hpp"
 #include "real_time_tools/thread.hpp"
 #include "kodlab_mjbots_sdk/abstract_realtime_object.h"
+#include "kodlab_mjbots_sdk/lcm_message_handler.h"
+
+namespace kodlab
+{
```

Comment:
Fix the brackets.

Reply:
Fixed in a3d7d629027fc762a059e3c3f2d441cbf60a1c7e